starter-code/iteration-1/main.ts

```diff
@@ -1,11 +1,32 @@
 // Iteration 1
 // 1. Create an array of strings
+let strings: Array<string> = [];
 
 // 2. Create a function to add a task in to the array, call it `addTask`, the function has to accept a string and don't have to return anything.
+function addTask(task:string):number{
+	strings.push(task);
+	console.log('It works');
+	return strings.length;
+}
+
+
+
 
 // 3. Create a function to list all tasks, it must show in the console de task.
+function listAllTasks(){
```

Comment:
this function should have a type of void ..
`function listAllTasks (): void {
     this.tasks.forEach((task, index) => {
      console.log(task);
  });
}`